<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PogChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="message_enter">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>370</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chat_window">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>40</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="log_out_button"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="log_out_button">
   <property name="text">
    <string>Выйти из учётной записи</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
